plat/intel/soc/stratix10/platform.mk

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (c) 2019, ARM Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+
+LOAD_IMAGE_V2 := 1
+
+PLAT_INCLUDES		:=	\
+			-Iinclude/plat/arm/common/			\
+			-Iinclude/drivers/synopsys/			\
+                        -Iinclude/plat/common/          		\
+			-Iinclude/plat/arm/common/aarch64/		\
```

Comment:
You have to remove ```include/plat/arm/common/aarch64``` and ```include/plat/arm/common```, you shouldn't use any ```plat/arm``` file. Also, remove the following lines:
```
			-Iinclude/drivers/synopsys/
```
and
```
			-Iinclude/common/tbbr				\
			-Iinclude/common/				\
			-Iinclude/drivers/partition			\
```
You have to use the full path for includes now.

This applies to the rest of the includes. You should try to compile with ``ERROR_DEPRECATED=1`` to see what you have to fix.